test/e2e/e2e-test-config.exapmle.yaml

```diff
@@ -60,7 +60,8 @@ defaultFeatures:
     - "nfd.node.kubernetes.io/worker.version"
     - "nfd.node.kubernetes.io/feature-labels"
   nodes:
-    my-node-1:
+    # NOTE: keys here are interpreted as regexps
```

Comment:
not part of the Patch but just noted that this file has a typo in the file name
`test/e2e/e2e-test-config.exapmle.yaml ` 
example...
can you fix this in this patch :smiley: